src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/MsQuicStream.cs

```diff
@@ -512,8 +512,17 @@ internal override void Shutdown()
         internal override int Read(Span<byte> buffer)
         {
             ThrowIfDisposed();
-
-            return ReadAsync(buffer.ToArray()).AsTask().GetAwaiter().GetResult();
```

Comment:
How did this ever work? :-) Or we just weren't testing it because we'd disabled all the stream conformance tests?

Reply:
I suspect the lather. I'll go through them and fix them. In some cases we simply return different exceptionalities (like QuicAbort Vs OperationCancelled) but there are still some suspects.